src/features/identityCheck/components/SomeAdviceBeforeIdentityCheckModal.tsx

```diff
@@ -10,40 +13,67 @@ import { IdCard } from 'ui/svg/icons/IdCard'
 import { Sun } from 'ui/svg/icons/Sun'
 import { IconInterface } from 'ui/svg/icons/types'
 import { ColorsEnum, getSpacing, Typo } from 'ui/theme'
+import { useNavigation } from '@react-navigation/native'
+import { UseNavigationType } from 'features/navigation/RootNavigator'
 
 interface Props {
   visible: boolean
   hideModal: () => void
 }
 
+const redirectUrl = `https://${env.WEBAPP_V2_DOMAIN}/verification-identite/fin`
```

Comment:
On a une variable `WEBAPP_V2_URL` :
```suggestion
const redirectUrl = `${env.WEBAPP_V2_URL}/verification-identite/fin`
```

Reply:
Elle n'est pas dans `env`, mais dans `import { WEBAPP_V2_URL } from 'libs/environment/useWebAppUrl'`